src/components/progressBar.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import './progressBar.css';
+
+const ProgressBar = () => <div role="progressbar" aria-valuenow="65" aria-valuemin="0" aria-valuemax="100" />;
+export default ProgressBar;
```

Comment:
Your efforts in adding the progress bar are commendable. However, to align with the suggested design, it would be better to use a percentage value inside the bar. This will not only make it look more like a progress indicator but also make your design more attention-grabbing for potential employers